general.css

```diff
@@ -0,0 +1,918 @@
+}
+
+.team-member-job {
+  margin: 0.5rem;
+  text-align: center;
+  letter-spacing: 0.08rem;
+  font-size: var(--paragraph-size);
+  font-style: normal;
+  color: var(--section-title-color);
+}
+
+.team-contact-icons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 0rem;
+  width: 100%;
+  padding: 1rem;
+  background: var(--first-font-color);
+  opacity: 0;
+  transition: opacity 500ms linear;
+}
+
+.team-image-container:active .team-contact-icons,
+.team-image-container:hover .team-contact-icons {
+  opacity: 0.6;
+}
+
+.portafolio-ilustration {
+  display: flex;
+  cursor: pointer;
+  width: 100%;
+  height: 100%;
+  background-color: var(--hover-color);
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  bottom: 0;
+  opacity: 0;
+  transition: opacity 500ms linear;
+}
+
+.team-image-container:active .portafolio-ilustration,
+.team-image-container:hover .portafolio-ilustration {
+  opacity: 0.9;
+}
+
+.portafolio-ilustration-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  transform: scale(0.8);
+  transition: transform 500ms linear;
+}
+
+.team-image-container:active .portafolio-ilustration-content,
+.team-image-container:hover .portafolio-ilustration-content {
+  transform: scale(1);
+}
+
+.portafolio-icon {
+  width: 3rem;
+  height: 3rem;
+  background: var(--first-font-color);
+  border-radius: 50%;
+  padding: 1rem;
+  margin: 1rem 0;
+}
+
+.icon-link {
+  display: inline-block;
+  width: 1rem;
+  height: 1rem;
+  stroke-width: 0;
+  stroke: currentColor;
+  fill: var(--hover-color);
+  transform: rotate(90deg);
+}
+
+.portafolio-ilustration-title{
+  color: var(--first-font-color);
+  font-size: var(--subtitle-size);
+  margin: 1rem;
+  margin-bottom: 0;
+}
+
+.portafolio-ilustration-subtitle {
+  color: var(--first-font-color);
+  font-size:var(--paragraph-size);
+  margin: 1rem;
+}
+
+.blog-comment {
+  border: 1px solid #f8f8f5;
+  margin: 0.5rem 1rem;
+  padding: 3rem;
+  border-radius: 3%;
+  background-position: top 0 left 0;
+  transition: background-position 5000ms ease;
+}
+
+.blog-comment-date {
+  font-size: var(--paragraph-size);
+  color: #8b8787;
+}
+
+.blog-comment-description {
+  font-size: var(--subtitle-size);
+  color: var(--section-title-color);
+  margin-bottom: 0;
+}
+
+.blog-comment-author {
+  font-size: var(--paragraph-size);
+  color: var(--section-title-color);
+}
+
+.blog-comment:active,
+.blog-comment:hover {
+  background: url("images/hover-image.jpg") bottom -5rem left 0 / auto 50vh no-repeat;
+}
+
+.blog-comment:active .blog-comment-description,
+.blog-comment:hover .blog-comment-description{
+  color: var(--hover-color);
+}
+
+.blog-comments {
+  display: flex;
+  flex-direction: column;
+}
+
+.question {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  border: 1px solid #f8f5f5;
```

Comment:
check the consistency of the colors, use hex for all of them, use variables for the colors